Deck descriptions

alexmiddeleer opened the pull request:
Allows users to add a deck description on the `/deck/:id` page

lily-pytel commented on next/pages/deck.js:
```diff
@@ -46,7 +46,9 @@ export default withRouter(({ router }) => {
   return (
     <Layout title={title} desc={description}>
-      <PageBanner image={PageBanner.IMG_DECKS} url="/decks">Decks</PageBanner>
+      <PageBanner image={PageBanner.IMG_DECKS} url="/decks">
```
LOL

lily-pytel commented on next/components/deck-description-edit.js:
```diff
@@ -0,0 +1,108 @@
+
+  const editButton = (
+    <div>
+      <style jsx>{`
```
I would personally add a bit of margin top here for some space

lily-pytel commented:
I don't know why the tests are failing because of stuff in another branch, but I would pull from the publish mode branch and make it the parent of this. I fixed some props warnings but it required some tests updates and stuff... somehow looks like Travis is taking it out on your branch

alexmiddeleer commented:
Yep no worries, I saw it failed after I rebased.

On Sun, Nov 24, 2019 at 7:21 PM lily-pytel <notifications@github.com> wrote:

> I don't know why the tests are failing because of stuff in another branch,
> but I would pull from the publish mode branch and make it the parent of
> this. I fixed some props warnings but it required some tests updates and
> stuff... somehow looks like Travis is taking it out on your branch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mythgard-hub/mythgard-hub/pull/237?email_source=notifications&email_token=ABJGQP26S4B2ZLACJJY7BYLQVMK63A5CNFSM4JRA4MYKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFAZDHA#issuecomment-557945244>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABJGQP7IIPU7XW2HHARMIO3QVMK63ANCNFSM4JRA4MYA>
> .
>
